src/hypertable_restrict_info.c

```diff
@@ -98,11 +98,12 @@ dimension_restrict_info_open_add(DimensionRestrictInfoOpen *dri, StrategyNumber
 	foreach (item, dimvalues->values)
 	{
 		Oid restype;
+		TimevalInfinity is_infinite;
```

Comment:
This is never checked so why is it part of the signature? I'm wondering whether macros would be more useful to check if an internal value is finite